about.css

```diff
@@ -0,0 +1,393 @@
+  width: 100%;
+  max-width: var(--max-width);
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  gap: 15px;
+  margin: auto;
+}
+
+.media-icon {
+  height: 13px;
+}
+
+.media-text {
+  color: #d3d3d3;
+  font-size: 11px;
+}
+
+.desktop-header {
+  width: 100%;
+  padding: 20px 20px;
+  box-shadow: 2px 2px 4px black;
+}
+
+.desktop-nav-container {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  gap: 40px;
+  max-width: var(--max-width);
+  margin: auto;
+}
+
+.desktop-nav-logo {
+  display: flex;
+  place-content: center;
+  margin-right: auto;
+}
+
+.desktop-nav-list {
+  display: flex;
+  align-items: center;
+  gap: 15px;
+}
+
+.close-btn {
+  position: absolute;
+  top: 20px;
+  cursor: pointer;
+}
+
+.nav-menu {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  gap: 50px;
+}
+
+.nav-logo {
+  min-width: 80%;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+}
+
+.list-item a {
+  color: var(--grey);
+  font-size: 25px;
+  font-weight: 500;
+}
+
+.active a {
+  color: var(--orange);
+  font-weight: 600;
+}
+
+.nav-campaign {
+  background-color: transparent;
+  border: 3px solid var(--orange);
+  color: var(--orange);
+  padding: 10px 20px;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.mobile-nav-campaign {
+  color: white;
+  background-color: #ec5242;
+  font-weight: 600;
+}
+
+/* ----------NAV - HAMBURGUER - FINISH------------- */
+
+.about-section {
+  padding: 90px 20px;
+  background-image: url(./assets/icons/about-background.svg);
+  background-repeat: no-repeat;
+  background-position: left top;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 7px;
+  background-color: #f7f7f7;
+}
+
+.nav-btn {
+  position: fixed;
+  top: 20px;
+  left: 20px;
+}
+
+.about-hello {
+  color: var(--orange);
+  font-size: 20px;
+  font-weight: 500;
+}
+
+.about-title {
+  width: 100%;
+  text-align: center;
+  color: var(--orange);
+  font-size: 27px;
+  letter-spacing: 1px;
+  font-weight: 900;
+  margin-bottom: 25px;
+  max-width: 900px;
+}
+
+.about-description {
+  color: var(--dark);
+  text-align: center;
+  background-color: white;
+  border: 2px solid #e2e1e1;
+  padding: 10px;
+  font-size: 12px;
+  line-height: 150%;
+}
+
+.contact-text {
+  color: var(--dark);
+  text-align: center;
+  font-size: 12px;
+  line-height: 150%;
+  font-weight: 600;
+}
+
+.email {
+  border-bottom: 1px solid var(--dark);
+  color: var(--dark);
+  font-weight: 600;
+  font-size: 12px;
+}
+
+.logo-section {
+  padding: 20px 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  border-bottom: 1px solid var(--grey);
+}
+
+.section-title {
+  font-weight: 400;
+  font-size: 18px;
+}
+
+.logo-description {
+  font-weight: 400;
+  font-size: 12px;
+  text-align: center;
+  color: #5f5f5f;
+}
```

Comment:
- [OPTIONAL] The `font-size` for all texts that are not headers in your design is small and it can cause accessibility issues for those who have eyesight challenges. To make the sizes of the font look decent, I would advise that you use `16px` as the value for font-size declarations in your stylesheets. 👍 😎